crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space/agent_info_update.rs

```diff
@@ -55,7 +55,7 @@ mod tests {
     async fn update_agent_info() {
         let (test_sender, _) = setup(SpaceInternalStub::new()).await;
 
-        tokio::time::timeout(Duration::from_millis(100), async {
+        tokio::time::timeout(Duration::from_millis(200), async {
```

Comment:
These tests were stable with the current timeouts but when I was running this with tarpaulin for coverage I was seeing these flake. The instrumented code runs much slower so that's expected. I'm increasing the times so I don't have to bother investigating the failures next time I want coverage from Kitsune tests.